macros/helpers/prime_line.cfg

```diff
@@ -80,16 +82,14 @@ gcode:
     G0 X{prime_line_x} Y{prime_line_y} F{St}
     G1 Z{line_height} F{Sz|int / 2}
 
-    # Add pressure in the nozzle
-    G92 E0
-    G1 E18 F300
-
```

Comment:
this should be either put in a different PR or created as option, so klippain users can decide if they want it or not. (default is "yes", to comply to old functionality if the variable is not defined)

Reply:
Makes sense to add this as a user variable, I'll work on adding those options.